test_client.py

```diff
@@ -13,7 +13,12 @@
 from ssb.packet_stream import PSClient, PSMessageType
 
 
-with open(os.path.expanduser('~/.ssb/secret')) as f:
+if 'SSB_DIR' in os.environ:
```

Comment:
What about:
```python
SSB_DIR = os.environ.get('SSB_DIR', os.path.expanduser('~/.ssb'))
```
?
I guess the cost of calling `expanduser` every time is negligible.